src/rp2_common/pico_crt0/crt0.S

```diff
@@ -23,16 +24,22 @@
 
 pico_default_asm_setup
 
+#ifdef PICO_NO_STORED_VECTOR_TABLE
+#warning PICO_NO_STORED_VECTOR_TABLE is no longer used. PICO_MINIMAL_STORED_VECTOR_TABLE is not identical but usually serves the same purpose
+#endif
+
 .section .vectors, "ax"
 .align 2
 
-.global __vectors, __VECTOR_TABLE
+.global __vectors, __VECTOR_TABLE, __vectors_end
 __VECTOR_TABLE:
 __vectors:
 .word __StackTop
 .word _reset_handler
-.word isr_nmi
```

Comment:
This seems to have removed the ability to hook NMI and HardFault without installing a RAM vector table.

Reply:
oops; goot catch... somehow the two paths looked the same to me!